frontend/src/utils/items.ts

```diff
@@ -431,3 +470,154 @@ export function getMediaStreams(
 ): MediaStream[] {
   return mediaStreams.filter((mediaStream) => mediaStream.Type === streamType);
 }
+
+/**
+ * Get the remote search results for an item and search params.
+ *
+ * @param item - The item to search for.
+ * @param searches - The search params to use.
+ * @returns - An array of remote search results.
+ */
+export async function getItemRemoteSearch(
+  item: BaseItemDto,
+  searches: IdentifySearchItem[]
+): Promise<RemoteSearchResult[] | undefined> {
+  const remote = useRemote();
+
+  const itemId = item.Id;
+
+  if (remote.sdk.api === undefined) {
+    return;
+  }
+
+  if (itemId === undefined) {
+    return;
+  }
+
+  // Split the query to `search-item-` and the rest to provider IDs.
```

Comment:
Would be great if you can add more context on why this is needed

Reply:
Updated the comment